inst/extdata/PA2022CH_fr_exec_summary/template.Rmd

```diff
@@ -97,7 +97,7 @@ number_peers <- readr::read_csv(
 
-Ce résumé est une synthèse des résultats individuels de votre portefeuille téléchargé \startunderscoreletter {`r portfolio_name`}\stopunderscoreletter. \startunderscoreletter `r paste0(toupper(substr(investor_name, 1, 1)), substr(investor_name, 2, nchar(investor_name)))` \stopunderscoreletter a reçu un résumé par portefeuille. Un rapport interactif individuel détaillé (par portefeuille ou groupé) peut être consulté via un accès personnalisé sur le site platform.transitionmonitor.com) en allemand, en français et partiellement en anglais. Un méta-rapport contenant des données anonymisées et agrégées de tous les participants, ainsi que des rapports sectoriels, est publié sur le site Internet de l'OFEV \textbf{(\href{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/climat-et-marche-financier.html}{lien})}.
+Ce résumé est une synthèse des résultats individuels de votre portefeuille téléchargé \startunderscoreletter {`r portfolio_name`}\stopunderscoreletter. \startunderscoreletter `r paste0(toupper(substr(investor_name, 1, 1)), substr(investor_name, 2, nchar(investor_name)))` \stopunderscoreletter a reçu un résumé par portefeuille. Un rapport interactif individuel détaillé (par portefeuille ou groupé) peut être consulté via un accès personnalisé sur le site platform.transitionmonitor.com en allemand, en français et partiellement en anglais. Un méta-rapport contenant des données anonymisées et agrégées de tous les participants, ainsi que des rapports sectoriels, est publié sur le site Internet de l'OFEV \textbf{(\href{https://www.bafu.admin.ch/bafu/fr/home/themes/climat/info-specialistes/climat-et-marche-financier.html}{lien})}.
```

Comment:
should `platform.transitionmonitor.com` be linkified?

Reply:
not according to the agreed upon templates